yew-functional-macro/tests/macro_tests.rs

```diff
@@ -0,0 +1,7 @@
+#[allow(dead_code)]
```

Comment:
Could you rename this to something like `functional_component_attr_test` instead? We might add more macros in the future.

Reply:
Did you just rename this back?